js/videoBlock.js

```diff
@@ -0,0 +1,51 @@
+const video = [
+    {
+        title: "The youngest Beatles tribute band",
+        frameborder: 0,
+        accelerometer: true,
+        encryptedMedia: true,
+        gyroscope: true,
+        pictureInPicture: true,
+        fullscreen: true,
+    },
+    {
+        title: "Teen Millionaires - successful businesses run by teens",
+        frameborder: 0,
+        accelerometer: true,
+        encryptedMedia: true,
+        gyroscope: true,
+        pictureInPicture: true,
+        fullscreen: true,
+    },
+]
+
+let source = ["https://www.youtube.com/embed/Uq-FOOQ1TpE", "https://www.youtube.com/embed/93mctUlDofw", "https://www.youtube.com/embed/1YCGM7FiCGg", "https://www.youtube.com/embed/vPukHebsQeQ", "https://www.youtube.com/embed/h11u3vtcpaY"]
+
+function displayVideos (video) {
```

Comment:
I can't see where do you call this function. So you need to execute it somewhere. 
And if you want it to have an argument it should be named `videos`, because you want to loop through them.

Reply:
True. Writing it at 1am shows :) Thanks